kairon/importer/validator/file_validator.py

```diff
@@ -279,12 +308,13 @@ def verify_utterances_in_stories(self, raise_exception: bool = True):
                 form_utterances.add(f"utter_ask_{form}_{slot}")
         utterance_actions = utterance_actions.difference(form_utterances)
 
-        for utterance in utterance_actions:
```

Comment:
any reason why this has been removed?

Reply:
Because i have collectively checked for utterances in stories and multiflow_stories.